src/routes/console/project-[project]/messaging/message-[message]/scheduleModal.svelte

```diff
@@ -0,0 +1,143 @@
+<script lang="ts">
+    import { invalidate } from '$app/navigation';
+    import { Modal } from '$lib/components';
+    import { Button, FormList, InputDate, InputTime, Helper } from '$lib/elements/forms';
+    import { addNotification } from '$lib/stores/notifications';
+    import { sdk } from '$lib/stores/sdk';
+    import { Submit, trackEvent, trackError } from '$lib/actions/analytics';
+    import { MessageStatus, MessagingProviderType, type Models } from '@appwrite.io/console';
+    import { Dependencies } from '$lib/constants';
+
+    export let show = false;
+    export let message: Models.Message & { data: Record<string, unknown> };
+    export let topics: Models.Topic[];
+
+    let now = new Date();
+    let timeZoneOffset: number;
+    let minDate: string;
+    // Use Sweden's locale (sv) since it matches ISO format
+    let date =
+        message.scheduledAt == null ? null : new Date(message.scheduledAt).toLocaleDateString('sv');
```

Comment:
`toISOString()` won't work because that converts it to UTC timezone, but i need the time to be local.

I could create a helper function in the date utils called...`toLocaleTimeInISOFormat()`. Ya?

Reply:
gotcha, yes that function sounds great 👍🏻